apps/deploy-web/src/components/sdl/RentGpusForm.tsx

```diff
@@ -298,13 +292,23 @@ export const RentGpusForm: React.FunctionComponent = () => {
           infoText={
             <Alert className="mb-4" variant="default">
               <p className="text-sm text-muted-foreground">
-                To create a deployment, you need to have at least <b>{minDeposit.akt} AKT</b> or <b>{minDeposit.usdc} USDC</b> in an escrow account.{" "}
+                {isManaged ? (
+                  <>
+                    To create a deployment, you need to have at least <b>${depositData?.min}</b> in an escrow account.{" "}
+                  </>
+                ) : (
+                  <>
+                    To create a deployment, you need to have at least <b>{minDeposit.akt} AKT</b> or <b>{minDeposit.usdc} USDC</b> in an escrow account.{" "}
+                  </>
+                )}
```

Comment:
This is repeated twice, perhaps we could have a hook that returns this message in both places